easy_thumbnails/widgets.py

```diff
@@ -54,9 +58,15 @@ def get_thumbnail(self, value):
             thumbnailer.thumbnail_storage = value.thumbnail_storage
         return thumbnailer.get_thumbnail(self.thumbnail_options)
 
-    def render(self, name, value, attrs=None):
-        output = super(ImageClearableFileInput, self).render(
-            name, value, attrs)
+    def render(self, name, value, attrs=None, renderer=None):
+        # Backward compatibility for Django < 1.11
```

Comment:
Instead of converting version strings back- and forward, I'd rather use

```
from django import VERSION as DJANGO_VERSION

...

if DJANGO_VERSION < (1, 11):
    ...
```

this prevents pulling in other libraries.